django/db/migrations/loader.py

```diff
@@ -109,11 +109,11 @@ def load_disk(self):
                 if was_loaded:
                     reload(module)
             self.migrated_apps.add(app_config.label)
-            migration_names = {
+            migration_names = [
```

Comment:
I considered making this `sorted()`, but I wanted to take the lightest possible touch for now, and not risk making a Django 6.1 upgrade any more difficult for anyone.